src/Text/Text.js

```diff
@@ -0,0 +1,19 @@
+import React, { forwardRef } from 'react'
+import classNames from 'classnames'
+
+// this component is not fully fleshed out, it's just here to add some typography
+function Text({ className, as, children }, ref) {
+  const Component = as ?? 'span'
```

Comment:
What is this magic? `const Component = as ?? 'span'`

Reply:
Haha, it's called [Nullish Coalescing](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator)

Basically I'm letting the user of `Text` use whatever component they want, so they could say:
```jsx
<Text as="p">Hello, World!</Text>
```
and the resulting element would be a `<p />`. If nothing is passed to `as`, it will default to using a `span`.